Trist-TVT-QuiteThePickle PR Review

TristGit opened the pull request:
**Mission Creation Checklist:**
- [X] Mission File
- [X] Ingame Play Area Markers
- [X] Ingame Briefings to all players
- [X] Side-based briefings
- [X] Objective Entities
- [X] Mission .conf file

**Faction(s):**
BLUFOR - QRF Air Assault
- RHS_USAF 

OPFOR - Helo Crash Interception Force
- CHDKZ 

INDFOR - USAF SpecOps
- RHS+WCS USAF

**Objectives/Win Conditions:**

BLUFOR must extract the INDFOR SpecOps squad either via air OR through alternative push-through land route.
OPFOR must locate the crash and prioritize elimination of the HV SpecOps assets.
INDFOR must fight/fade/evade the OPFOR while using alternative means to communicate their whereabouts to BLUFOR

**Terrain:**

Eden; Rocky mountainside/Coastline

**Short mission description:**

A SpecOps (INDFOR) squad RTBing from a mission has been shot down, and has no means of comms to relay their status to BLUFOR but they have greenlit a rapid air assault sortie to retrieve the unit. OPFOR intends to eliminate the remnants of the crash and prevent extraction. INDFOR must think tactically and weigh their options in flagging down BLUFOR and be ready for OPFOR to fully commit to stopping their extraction once their reveal their position.

**Slotting Ratio:**

1:2 / 33:60:8

**Time limit:**

50 minutes

**Mission Briefing Screenshot:**

![fa6d04265397e45e52fe87cd7a8625bd](https://github.com/user-attachments/assets/21ebf8b4-496f-45f7-87d1-13b20023e3c1)



**Design concept/thoughts:**

Overall mission design is to have an element that is closer and at risk than the rest of the main force, of course using INDFOR for the cut off force is most logical. The SpecOps IND squad is equipped with normal and Star green flares to help them communicate with any Blufor, while having to remain moderately low profile until they are ready. OPFOR is less heavier equipped overall but more numerous and has land vehicles. If the chinook gets destroyed or if its generally a better idea BLU/IND can spearhead through OPF forces. IND has a head start to move while remaining stealthy, OPFOR has vehicles to catch up but has to be vigilant , BLUFOR must identify- land- and secure them or aid in any fighting.

Terrain involved also is a longer strip of land with only coastline/keyholes of potential LZ's. My intention is for the fight to be over the safety of the LZ and promote the IND/BLU forces to meet and fight for the LZ. This mission in theory can end rather quick if IND/BLU act quick enough so that may be a downside but I wanted the terrain to promote difficulty with that.

Tried my best for asymmetry.

**Other Notes:**
- I do NOT have any scripts in play for the rescue routes to signal the presence of IND/BLU that they have extracted (I just dont have the scripting xp)

- Most Importantly: _**MY GITHUB DT DID SOME FUCKYWUCKY SHIT WITH THE LAST FEW COMMITS YOU/HARRY HAVE MADE TO THINGS FOR SOME REASON? CAN YOU PLEASE VERIFY THIS WONT OVERWRITE ANYTHING, not a ton of Git xp either to identify that.**_

- Let me know whats up and what I have to change. Trying to make more unique stuff.

HarryEngineering commented:
Seems like you have some merge branch issues. It's changing way more files than just your mission files. Likely rolling back a change errantly.

TristGit commented:
Thanks Harry. Fixed. File mismatch was due to me accidentally reverting changes since the commit looked strange. 
Also I [asked in dev](https://discord.com/channels/237991125523103747/468879945293234177/1357209335612047502) if GS's allowed easily applying helmet/mask/nvg combos instead of making tons of prefabs, so that's why I have handfuls of variants of helmets with the PR.

tanaaKa commented:
QA feedback:

- Reorganize the SF team to actually be a SF team. Include roles that allow them to be specialized such as marksman, MAT gunner, etc... They should be as light weight as possible which means no ARs or AARs. Consider 4x optics as well for them (optional due to power creep).
- Upgrade the NVGs for SF to AN/PVS-31s rather than AN/PVS-14s
- OPFOR does not have enough ground transport for a company sized element
- BLU callsigns are incorrect:
![image](https://github.com/user-attachments/assets/618db865-9ccc-4f7a-86d0-71141a604904)
- Bump mission time up to 60 mins
- Enable radio espionage in the game mode settings so teams need to use callsigns/code words
- DISCRETIONARY: Add MS2000 marker and HEL6 strobes to SF if you feel as though they aren't going to negate the flare aspect or make it too easy for BLU to find them.
- DISCRETIONARY: Remove two medics and add an unarmed mi8 helicopter to search for the SF team 
- DISCRETIONARY: Do you want IND and BLU to share radio freqs? You can make this happen by setting both of their encryption keys to the same phrase in the faction manager:
![image](https://github.com/user-attachments/assets/1e9f89a3-f1ee-4d3b-9b68-969bb6835185)
- DISCRETIONARY: The SF team is visible to blufor. Is this okay?
![image](https://github.com/user-attachments/assets/67fd0435-b311-4402-bb9c-7260b1e8ca96)
- DISCRETIONARY: If you REALLY want to make this an exciting mission where one team can't completely dominate the other, establish specific LZs/Extraction zones that all teams know about. This will empower the need for code words to prevent espionage and empower OPFOR to commit espionage rather than searching blindly.

In general I think the mission is too open ended. There's entirely situations where opfor spreads too thin across the AO, very few have contact and are destroyed by a lack of numbers, and IND simply escapes with a bored blufor.

tanaaKa commented:
I've added a trigger for IND extract. Check the example:
![image](https://github.com/user-attachments/assets/2cc3c2fc-de40-46b3-8bbd-661cbc8d400f)

# IF YOU CHANGE THE INDFOR COMPOSITION
Change the trigger prefab filter as noted in the screenshot. Otherwise there's a chance the trigger doesn't fire.

TristGit commented:
> QA feedback:
> 
> * Reorganize the SF team to actually.....
@tanaaKa 


- [X] Changed SF (PVS31s, Marksman w/MK14, MAT, Gren.), Changed OPF Added +2 Urals
- [X] Mission Parameters: Espionage enabled, time set to 60 minutes
- [X] Fixed Callsigns


Discretionary:
- [X] Removed 2 Medics - Added 2 Pilots/Mi8 to OPF 
- [ ] Didnt add any markers or strobes to SF due to potential of not being able to avoid the marker to BLU
- [ ] **Have NOT considered/didnt know that IND/BLU see group markers**, not linking their radio freqs for some sort of attempt of signaling/locating if no way behind this, Feel free to make changes that are more sensible **involving the group markers/sides being able to use vehicles together as allies issue**
  _**Feel free to delete and/or change anything related to the markers showing**_ 
- [X] Added 6 Potential LZ's / Added Mission: Rules briefing to explain things surrounding LZ's
  _**Feel free to delete a few and/or change the LZ's (see new mission image)**_ 
- [X] Added 2nd Trigger to Land Route (And changed filters)

I guess double check for any fuckups as well
